lnrpc/routerrpc/forward_interceptor.go

```diff
@@ -194,3 +166,54 @@ func (r *forwardInterceptor) resolveFromClient(
 		)
 	}
 }
+
+func (r *forwardInterceptor) handleFailMessage(failMessage []byte,
```

Comment:
does this function have to be on the `forwardInterceptor`?

Reply:
good point, moved validation to interceptable switch. it goes deeper than just a bit of input checking.